src/SFA.DAS.Payments.Audit.DatalockService/Infrastructure/Ioc/AuditDataLockEventServiceModule.cs

```diff
@@ -40,12 +43,23 @@ protected override void Load(ContainerBuilder builder)
                 .As<IHandleMessageBatches<DataLockEventModel>>()
                 .InstancePerLifetimeScope();
 
+            builder.Register(c =>
```

Comment:
Seems like a lot of replicated code here, I think this registration could be added to the common audit autofac module in SFA.DAS.Payments.Audit.Application.Infrastructure.Ioc.AuditModule instead.